infrastructure/template.yaml

```diff
@@ -177,6 +178,9 @@ Resources:
       EvaluationPeriods: 1
       Threshold: 1
       ComparisonOperator: "GreaterThanOrEqualToThreshold"
+      AlarmActions:
+        - !Ref AlarmNotificationTopic
+      ActionsEnabled: true
```

Comment:
Cool, does this mean we could set up a bunch of these on all envs, and then do one additional PR to switch them "on" when we're ready for prod? Avoiding any dev noise in the meantime?

Reply:
Yeah I think that must be what this flag is designed for